### DISCUSSION TITLE
post/memo/nextjs-fetch-failed

### DISCUSSION THREAD

**Discussion by giscus** [kyechan99/kyechan99.github.io#24] [General]
# post/memo/nextjs-fetch-failed

Error occurred prerendering page, TypeError fetch failed

https://kyechan99.github.io/post/memo/nextjs-fetch-failed

**Comment by youngwan2** [comment]
진짜 감사하다는 말씀 꼭 전하고 싶어서 댓글 남깁니다.. gpt, 바드, 코드 파일럿 등등 부터 해서, next.js 공식 깃허브 이슈, 타블로그 등등에서도 해결 방법을 찾지 못해서 끙끙거리고 있었는데, 개발자님 덕분에 해결 할 수 있었습니다.. 진짜 이렇게 기록을 남겨 주시지 않았다면 하루 종일 왜 안 되는거지 하면서 머리가 터지고 있었을 겁니다. 정말 감사합니다.

**Comment by rldbs2002** [comment]
저도 정말로 감사하다는 말 드립니다. nextjs13에서 getServerSideProps를 안쓰고 직접적으로 fetch해서 사용한다하여 코드를 작성했지만 실제 npm run build를 하거나 deploy할때 fetch에러가 나서 서버컴포넌트임에도 불구하고 useEffect를 써서 CSR처리를했는데 이렇게하니 에러가 해결되었네요 정말 감사합니다...
